webapp/src/ts/services/enketo.service.ts

```diff
@@ -135,25 +135,6 @@ export class EnketoService {
           $element.text(this.translateService.instant('enketo.' + $element.attr('data-i18n')));
         });
 
-        // TODO remove this when our enketo-core dependency is updated as the latest
-        //      version uses the language passed to the constructor
-        const languages = $html.find('#form-languages option');
-        if (languages.length > 1) { // TODO how do we detect a non-localized form?
-          // for localized forms, change language to user's language
-          $html
-            .find('[lang]')
-            .removeClass('active')
-            .filter('[lang="' + language + '"], [lang=""]')
-            .filter((idx, element) => {
-              // localized forms can support a short and long version for labels
-              // Enketo takes this into account when switching languages
-              // https://opendatakit.github.io/xforms-spec/#languages
-              return !$(element).hasClass('or-form-short') ||
-                ($(element).hasClass('or-form-short') && $(element).siblings( '.or-form-long' ).length === 0 );
-            })
-            .addClass( 'active' );
-        }
```

Comment:
This code is from https://github.com/enketo/enketo-core/blob/4.41.6/src/js/language.js#L53-L62  
We don't need it anymore, we now trigger a change on the `#form-languages` which calls the highlighted code in enketo-core